examples/1d/parameters_source_terms.jl

```diff
@@ -0,0 +1,58 @@
+# TODO: Taal refactor, rename to
+# - euler_source_terms.jl
+# or something similar?
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations1D(1.4)
+
+initial_conditions = initial_conditions_convergence_test
+
+surface_flux = flux_lax_friedrichs
+solver = DGSEM(4, surface_flux)
+
+coordinates_min = (0,)
+coordinates_max = (2,)
```

Comment:
It would be great if we can have this as scalars. But there already might be an issue for it...

Reply:
Fixed in 3d1c6f91dd8f3d9f5dcfec4ddebf2cb10b249a1d.